C/plugins/storage/sqlite/connection.cpp

```diff
@@ -711,8 +711,6 @@ SQLBuffer	jsonConstraints;
 									sql.append("strftime('%Y-%m-%d %H:%M:%f', ");
 									sql.append((*itr)["column"].GetString());
 									sql.append(", 'utc')");
-									sql.append(" AS ");
```

Comment:
Don't we need this to get the right column names back?

Reply:
I had problems at the first run 
adding a log, I found there was  .. AS field... twice in the created SQL,
as (at my understanding) it was already added by the code : 

{code}
if (itr->HasMember("alias")) {
    sql.append(" AS \"");
    sql.append((*itr)["alias"].GetString());
{code}

the current code works fine for the tests I have done (related to this topic) 
but I would like to be sure if it is in general fine or 
if there are considerations to do that I don't see right now because they require a broader view.